src/storage/DataAccessorBasedStore.ts

```diff
@@ -97,6 +123,21 @@ export class DataAccessorBasedStore implements ResourceStore {
   public async getRepresentation(identifier: ResourceIdentifier): Promise<Representation> {
     this.validateIdentifier(identifier);
 
+    if (this.metadataStrategy.isAuxiliaryIdentifier(identifier)) {
+      const subjectIdentifier = this.metadataStrategy.getSubjectIdentifier(identifier);
+      const metadata = await this.accessor.getMetadata(subjectIdentifier);
+      if (isContainerPath(metadata.identifier.value)) {
+        // Add containment triples of non-auxiliary resources
+        for await (const child of this.accessor.getChildren(subjectIdentifier)) {
+          if (!this.auxiliaryStrategy.isAuxiliaryIdentifier({ path: child.identifier.value })) {
+            metadata.add(LDP.terms.contains, child.identifier as NamedNode);
+          }
+        }
+      }
+      this.removeResponseMetadata(metadata);
```

Comment:
Some more context on why the graph solution was chosen instead of having every component keep track of which metadata it added so it can remove it again: #782, #790.

I was first thinking of removing the graph but the above PRs have several arguments why that solution was preferred.

One of my issues with my suggested solution is that this introduces extra knowledge into the `QuadToRdfConverter`.

Reply:
As an exercise, I currently added such a transformer to QuadToRdfConverter. 
When you have this, could a solution be adding a field to the preferences that indicate whether you want to keep the response metadata (transformed to the default graph).
In the `ConvertingPatcher` the preference of not having this response metadata can be added, which means that it then will not be written away.

Reply:
I would not add that for now but add an issue afterwards that we probably want to look into a more robust solution.

Patching should still work after doing this change, since the output representation when requesting metadata is `internal/quads` and the `ConvertingPatcher` wants to convert to `internal/quads`, meaning no conversion is going to take place (which is one of the reasons I suggested returning quads instead of turtle when requesting metadata).